src/main/java/mate/academy/ConcurrentMergeSortExample.java

```diff
@@ -11,6 +11,7 @@ public static void sort(int[] array) {
 
     public static void main(String[] args) {
         int[] array = {12, 11, 0, 13, 5, 6, 7, -1};
```

Comment:
It's a good practice to use more descriptive variable names. Instead of 'array', a name like 'unsortedArray' or 'inputArray' might give more context.